reactivesocket-tck-drivers/src/test/resources/client$.txt

```diff
@@ -1,262 +1,6 @@
-subscribe%%rs%%1f8ef22d-ca4c-4576-b5f4-bc42b19ab36d%%c%%d
-request%%1%%1f8ef22d-ca4c-4576-b5f4-bc42b19ab36d
-await%%terminal%%1f8ef22d-ca4c-4576-b5f4-bc42b19ab36d
-assert%%received_n%%1f8ef22d-ca4c-4576-b5f4-bc42b19ab36d%%1
-assert%%no_error%%1f8ef22d-ca4c-4576-b5f4-bc42b19ab36d
-assert%%completed%%1f8ef22d-ca4c-4576-b5f4-bc42b19ab36d
-assert%%received%%1f8ef22d-ca4c-4576-b5f4-bc42b19ab36d%%jimbo,jones
-!
-name%%requestStreamEmpty
-pass
-subscribe%%rs%%6443bde0-7d29-45ab-a2f8-a21a5d61e6d3%%a%%b
-request%%1%%6443bde0-7d29-45ab-a2f8-a21a5d61e6d3
-await%%terminal%%6443bde0-7d29-45ab-a2f8-a21a5d61e6d3
-assert%%completed%%6443bde0-7d29-45ab-a2f8-a21a5d61e6d3
-assert%%received_n%%6443bde0-7d29-45ab-a2f8-a21a5d61e6d3%%0
-assert%%no_error%%6443bde0-7d29-45ab-a2f8-a21a5d61e6d3
-!
-name%%requestResponseInterleave
-pass
-subscribe%%rr%%9eff90c8-a2b1-4073-8111-2079d3814a0d%%i%%j
-subscribe%%rr%%666298d0-8ffe-4033-b633-f957b2d15e6c%%k%%l
-request%%1%%9eff90c8-a2b1-4073-8111-2079d3814a0d
-subscribe%%rr%%7675f575-323a-4292-9e6b-aa16833826df%%m%%n
-request%%1%%7675f575-323a-4292-9e6b-aa16833826df
-await%%atLeast%%9eff90c8-a2b1-4073-8111-2079d3814a0d%%1%%100
-request%%1%%666298d0-8ffe-4033-b633-f957b2d15e6c
-await%%atLeast%%666298d0-8ffe-4033-b633-f957b2d15e6c%%1%%100
-await%%atLeast%%7675f575-323a-4292-9e6b-aa16833826df%%1%%100
-assert%%received%%9eff90c8-a2b1-4073-8111-2079d3814a0d%%homer,simpson
-assert%%received%%666298d0-8ffe-4033-b633-f957b2d15e6c%%bart,simpson
-assert%%received%%7675f575-323a-4292-9e6b-aa16833826df%%seymour,skinner
-!
-name%%requestResponseCancel
-pass
-subscribe%%rr%%8f94f232-7f43-4641-8594-3af58a065f2c%%g%%h
-cancel%%8f94f232-7f43-4641-8594-3af58a065f2c
-assert%%canceled%%8f94f232-7f43-4641-8594-3af58a065f2c
-assert%%no_error%%8f94f232-7f43-4641-8594-3af58a065f2c
-assert%%no_completed%%8f94f232-7f43-4641-8594-3af58a065f2c
-assert%%received_n%%8f94f232-7f43-4641-8594-3af58a065f2c%%0
-!
-name%%requestResponseTimeoutFail
-fail
-subscribe%%rr%%79828520-26e1-407b-8c52-05ff4b3d232f%%e%%f
-request%%1%%79828520-26e1-407b-8c52-05ff4b3d232f
-await%%terminal%%79828520-26e1-407b-8c52-05ff4b3d232f
-!
-name%%requestResponseError
-pass
-subscribe%%rr%%e2284394-ecad-4d82-8c30-6e9be225a793%%c%%d
-request%%1%%e2284394-ecad-4d82-8c30-6e9be225a793
-await%%terminal%%e2284394-ecad-4d82-8c30-6e9be225a793
-assert%%received_n%%e2284394-ecad-4d82-8c30-6e9be225a793%%0
-assert%%no_completed%%e2284394-ecad-4d82-8c30-6e9be225a793
-assert%%error%%e2284394-ecad-4d82-8c30-6e9be225a793
-!
-name%%requestResponsePass
-pass
-subscribe%%rr%%0e65a66f-0322-41e1-925c-b9e55b0db7f4%%a%%b
-request%%1%%0e65a66f-0322-41e1-925c-b9e55b0db7f4
-await%%atLeast%%0e65a66f-0322-41e1-925c-b9e55b0db7f4%%1%%100
-assert%%no_error%%0e65a66f-0322-41e1-925c-b9e55b0db7f4
-assert%%completed%%0e65a66f-0322-41e1-925c-b9e55b0db7f4
-assert%%received%%0e65a66f-0322-41e1-925c-b9e55b0db7f4%%a,a
+subscribe%%fnf%%746757d6-8e87-4673-b000-34e7fb305c33%%a%%b
+request%%1%%746757d6-8e87-4673-b000-34e7fb305c33
```

Comment:
Oops sorry, I was just deleting some parts of the script so I could run a single test. I can look into modifying the driver so that you can choose what test to run